src/classes/AccountDeDuper.cls

```diff
@@ -1,7 +1,7 @@
-public with sharing class AccountDeDuper {
+public with sharing class Account_DeDuper {
 
   // Constructor.
-Public AccountDeDuper () {
+Public AccountDe_Duper () {
 
 }
 Public List<Account> SearchAccount (Account pCurrentRecord, String pAccountName, String pPostCode, String pCity, id pRecordTypeId, Boolean pShowAllRecordTypes, Integer pSearchLimit) {
```

Comment:
* This method has a very long parameter list. I suggest to encapsulate many related parameters into a single object, or consider passing some parameters in the constructor prior to the invocation of this method.
* Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.